### DISCUSSION TITLE
PnP filter webpart does not present any filters for the defined RefinableStrings.

### DISCUSSION THREAD

**Discussion by ghost** [microsoft-search/pnp-modern-search#3284] [Q&A] [answered]
I am working a lot with the PnP Search webpart, Including the Search Box, Search Result, Search verticals and the Search filter webpart. For a customer I am creating a Search Center where the goals is to be able to search and filter on specific presentations. So I have created a Content type (VO-PowerPoint) in the Content Type hub, with 3 extra columns (also defined in the Content Type Hub):
- Language1 (Choice field with the values English, Dutch, French and German)
- Sector (Managed Metadata field, with Taxonomy defined on tenant level)
- Service1 (Choice field with multiple values defined)

After that, I uploaded a number of presentations with this contenttype and added the required metadata.
As a final step, I defined 3 RefinableStrings on tenant level to create the filter later on:
- RefinableString02
- RefinableString03
- RefinableString04

![image](https://github.com/microsoft-search/pnp-modern-search/assets/54032115/cda5f92b-57a3-4c48-81b6-f3c33a6808e0)

Final step before defining the filter is to select The RefinableStrings as Properties:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/54032115/76fee8b4-1804-493f-b48c-44d5bccde43a)

And check if they are actually filled:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/54032115/5f3ed094-176a-43a0-a9ea-26fa02d989d6)

All good so far. But when I configure the PnP filter webpart, i can select the RefinableStrings:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/54032115/bfb99db1-9f2f-4f7a-868f-b2aee388cd4f)

But no filter is actually presented, although the presentation are there:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/54032115/72bc39d5-ca08-4918-a9fd-c49159f86610)

Already tried to create new refinableStrings. But no success.  Using the newest version of the webparts.

All help is appreciated! Struggling with this for days now.

**Comment by kasperbolarsen** [comment, marked as answer]
Hi, I would say that the most likely reason is that the Results web part does not point to the Refiner web part  and/or the Refiner Web part does not point to the Results web part ( on tab no 3 on both web part).
Can you check this ?

**Reply by ghost** [reply]
Hi, thanks for the quick reply. I checked this, and it looks all good to me....

![image](https://github.com/microsoft-search/pnp-modern-search/assets/54032115/e1ef5b94-9468-4d68-a8c4-aa720fb892f6)

![image](https://github.com/microsoft-search/pnp-modern-search/assets/54032115/0252887f-c897-4bbc-989e-04b06b2ed010)

**Reply by ghost** [reply]
Apologies @kasperbolarsen, I completely forgot to link it also the other way around! I must link the result webpart to the filter webpart as well. Thanks!
